tests/schema/hooks.test.ts

```diff
@@ -238,11 +234,7 @@ it('in dev stage, warns if a wrong arg named is passed to ordering', async () =>
     await generateSchemaAndTypes(datamodel, [Query, User])
   })
 
-  expect($output).toMatchInlineSnapshot(`
-    "
-    Warning: Your GraphQL \`Query\` object definition is projecting a relational field \`users\`. On it, you are declaring that clients be able to order by Prisma \`User\` model field \`unknownField\`. However, your Prisma model \`User\` model has no such field \`unknownField\`
-    "
-  `)
+  expect($output).toMatchInlineSnapshot(`""`)
```

Comment:
These snapshot updates look wrong?

Reply:
I didn't write any of this test code or make changes to proxification, but I did manage to screw up a merge conflict :man_facepalming: Resolved!